### PR TITLE
Update team Github Page and README.md

### PR DESCRIPTION
**Pull request by Criss-Wang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2021S1-CS2103-W14-4/tp/actions)
```
Comment (zhengweii) [inline README.md]: Might need to change the URL for the badge!

File: README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
+* This is **software engineering project done by Team W14-4 from CS2103**.<br>
+  The project aims to:
+  * help medical supply managers keep track of medical products and storage
+  * allow fast typists to reap the benefit of swift user interaction with CLI and GUI
 * The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
```
Comment (qlchan24) [inline README.md]: Addressbook can be changed to CLI-nic

File: README.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
   * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
   * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+* It is named `CLI-nic` because it emphasizes on user interaction via CLI and serve to smoothen the clinical product's supply flow.
```
Comment (qlchan24) [inline README.md]: 'serves'

File: docs/index.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2021S1-CS2103-W14-4/tp/actions)
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**CLI-nic is a desktop application to help medical supply managers keep track of products and storage.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
 
 * If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
```
Comment (qlchan24) [inline docs/index.md]: name changed

File: docs/index.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
 * If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
```
Comment (qlchan24) [inline docs/index.md]: name changed

File: docs/index.md [hunk 6]
```diff
@@ -1,14 +1,16 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: CLI-nic
 ---
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2021S1-CS2103-W14-4/tp/actions)
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
```
Comment (jeffreytjs) [inline docs/index.md]: To check with prof regarding the changes to be made to monitor code coverage

**Review by tohyuting — APPROVED** [review]
LGTM

**Pull request merged**